services/qs-spark-service/src/main/java/ai/quantumics/api/repo/AwsConnectionRepo.java

```diff
@@ -18,5 +18,6 @@ public interface AwsConnectionRepo extends JpaRepository<AWSDatasource, Integer>
 	Optional<List<AWSDatasource>> findByActiveOrderByCreatedDateDesc(boolean active);
 
 	Optional<AWSDatasource> findByConnectionNameIgnoreCase(String connectionName);
+	List<AWSDatasource> findByActiveAndConnectionNameStartingWithIgnoreCaseOrActiveAndConnectionNameEndingWithIgnoreCase(boolean active, String connectionName, boolean active1, String connectionName1);
```

Comment:
Is it only starting and ending? what about inbetween

Reply:
As per requirement, we only need to consider starts with or ends with